rhscl-mariadb-apb/apb.yml

```diff
@@ -0,0 +1,43 @@
+id: a8e652c1-11e0-4202-b6df-09517f75ef9a
+name: rhscl-mariadb-apb
+image: registry.access.redhat.com/rhscl/mariadb-101-rhel7
```

Comment:
This should be the name of the APB image. So since we are going to publish this to dockerhub at ansibleplaybookbundle you would want ```image: ansibleplaybookbundle/rhscl-mariadb-apb```